client/blocks/app-promo/docs/example.jsx

```diff
@@ -0,0 +1,7 @@
+import AppPromo from '..';
+
+export default function AppPromoExample() {
+	return <AppPromo />;
```

Comment:
On a side note, this example isn't too useful 😅 I wonder if we should just remove it.